src/components/molecules/ChatsList/ChatsList.tsx

```diff
@@ -153,10 +154,10 @@ const ChatsList: React.FunctionComponent = () => {
               const sender = { ...chatUsers![userId], id: userId };
               const lastMessageExchanged =
                 discussionPartnerWithLastMessageExchanged?.[userId];
+              const isUnreadMessage = !lastMessageExchanged.isRead;
               const profileName = sender.anonMode
                 ? DEFAULT_PARTY_NAME
                 : sender.partyName;
-
```

Comment:
Did `prettier` delete this? I would have expected it to keep a line here.

Reply:
I'm not sure. But even if it was user deleted the prettier should fix it.

Reply:
Not to distract us from the prize of getting this merged, but I believe prettier isn't opinionated about 0/1 line, only when it's >1.